src/index.jsx

```diff
@@ -24,7 +24,7 @@ if (isClient) {
   let { pathname } = window.location;
   let trimmed = pathname.replace(/(.+)\/$/, '$1');
   let entryPage = findInContent(Content, item => item.url === trimmed);
-  let entryPath = entryPage.path.replace('src/content/', '');
+  let entryPath = entryPage && entryPage.path.replace('src/content/', '');
```

Comment:
Why this change?

Reply:
`/vote`, `/organization` and `starter-kits` are not in the content tree based on discovering markdown files in the file system. This causes `entryPage` to be undefined when visiting any of those pages, which then throws on `undefined.path.replace()`

Reply:
Correct